Base/Logic/vtkSlicerApplicationLogic.cxx

```diff
@@ -296,6 +298,60 @@ void vtkSlicerApplicationLogic::SetMRMLSceneDataIO(vtkMRMLScene* newMRMLScene,
       remoteIOLogic->AddDataIOToScene();
       }
     }
+
+  // Setting Orientation Matrices presets
+  vtkNew<vtkMatrix4x4> axialSliceToRAS;
+  axialSliceToRAS->SetElement(0, 0, -1.0);
+  axialSliceToRAS->SetElement(1, 0,  0.0);
+  axialSliceToRAS->SetElement(2, 0,  0.0);
+  axialSliceToRAS->SetElement(0, 1,  0.0);
+  axialSliceToRAS->SetElement(1, 1,  1.0);
+  axialSliceToRAS->SetElement(2, 1,  0.0);
+  axialSliceToRAS->SetElement(0, 2,  0.0);
+  axialSliceToRAS->SetElement(1, 2,  0.0);
+  axialSliceToRAS->SetElement(2, 2,  1.0);
+
+  vtkNew<vtkMatrix4x4> sagittalSliceToRAS;
+  sagittalSliceToRAS->SetElement(0, 0,  0.0);
+  sagittalSliceToRAS->SetElement(1, 0,  -1.0);
+  sagittalSliceToRAS->SetElement(2, 0,  0.0);
+  sagittalSliceToRAS->SetElement(0, 1,  0.0);
+  sagittalSliceToRAS->SetElement(1, 1,  0.0);
+  sagittalSliceToRAS->SetElement(2, 1,  1.0);
+  sagittalSliceToRAS->SetElement(0, 2,  1.0);
+  sagittalSliceToRAS->SetElement(1, 2,  0.0);
+  sagittalSliceToRAS->SetElement(2, 2,  0.0);
+
+  vtkNew<vtkMatrix4x4> coronalSliceToRAS;
+  coronalSliceToRAS->SetElement(0, 0, -1.0);
+  coronalSliceToRAS->SetElement(1, 0,  0.0);
+  coronalSliceToRAS->SetElement(2, 0,  0.0);
+  coronalSliceToRAS->SetElement(0, 1,  0.0);
+  coronalSliceToRAS->SetElement(1, 1,  0.0);
+  coronalSliceToRAS->SetElement(2, 1,  1.0);
+  coronalSliceToRAS->SetElement(0, 2,  0.0);
+  coronalSliceToRAS->SetElement(1, 2,  1.0);
+  coronalSliceToRAS->SetElement(2, 2,  0.0);
+
+  // Setting a Slice Default Node
+  vtkMRMLSliceNode* defaultNode = vtkMRMLSliceNode::SafeDownCast
+      (newMRMLScene->GetDefaultNodeByClass("vtkMRMLSliceNode"));
+  if (defaultNode)
+    {
+    defaultNode->AddSliceOrientationPreset("Axial", axialSliceToRAS.GetPointer());
+    defaultNode->AddSliceOrientationPreset("Sagittal", sagittalSliceToRAS.GetPointer());
+    defaultNode->AddSliceOrientationPreset("Coronal", coronalSliceToRAS.GetPointer());
+    }
+  else
+    {
+    defaultNode = vtkMRMLSliceNode::SafeDownCast
+        (newMRMLScene->CreateNodeByClass("vtkMRMLSliceNode"));
+    defaultNode->AddSliceOrientationPreset("Axial", axialSliceToRAS.GetPointer());
+    defaultNode->AddSliceOrientationPreset("Sagittal", sagittalSliceToRAS.GetPointer());
+    defaultNode->AddSliceOrientationPreset("Coronal", coronalSliceToRAS.GetPointer());
+    newMRMLScene->AddDefaultNode(defaultNode);
+    defaultNode->Delete(); // scene owns it now
+    }
```

Comment:
changing in taht way, it gives this error in compilation. I don't know why

vtkMRMLScene (0x2c0f1e0): vtkMRMLScene::AddDefaultNode failed: invalid node